### DISCUSSION TITLE
[Question]: Is the base Docker image going to be upgraded to Debian 12?

### DISCUSSION THREAD

**Discussion by remz1337** [blakeblackshear/frigate#11842] [Ask A Question]
### What is your question:

I noticed that the base image used when building Frigate is Debian 11. Is there any plans on upgrading to Debian 12?

**Comment by NickM-27** [comment]
Updating to a new base image is extremely involved. It affects everything that is used including a newer version of python, newer drivers for hardware acceleration, newer drivers for object detection and corals. It is planned but will be a long road and require a major update.

**Reply by remz1337** [reply]
Definitely. I did not meant to say it was a small task. Since Debian 11 is going to be EOL next month (july 2024), I was wondering how much of a priority this is for Frigate.
